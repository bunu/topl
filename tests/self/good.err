TOPL: A new Treap was computed on the basis of an older-than-current one.
  -> start: 
  -> start: call_insert(obj[0],"Radu")
  -> start: call_insert(obj[0],-1188957731,"Radu")
  -> start: return_insert(obj[1])
  -> current: return_insert(obj[1])
  -> old: call_insert(obj[1],"Dino")
  -> error: call_insert(obj[1],1018954901,"Dino")
TOPL: A new Treap was computed on the basis of an older-than-current one.
  -> start: 
  -> start: call_insert(obj[0],"Radu")
  -> start: call_insert(obj[0],-1188957731,"Radu")
  -> current: return_insert(obj[1])
  -> current: return_insert(obj[1])
  -> old: call_insert(obj[1],"Dino")
  -> error: call_insert(obj[1],1018954901,"Dino")
TOPL: A new Treap was computed on the basis of an older-than-current one.
  -> start: return_insert(obj[0])
  -> start: return_insert(obj[0])
  -> start: call_insert(obj[0],"Dino")
  -> start: call_insert(obj[0],1018954901,"Dino")
  -> start: call_insert(obj[1],1018954901,"Dino")
  -> start: return_insert(obj[2])
  -> start: return_insert(obj[3])
  -> current: return_insert(obj[3])
  -> old: call_insert(obj[3],"Grigore")
  -> error: call_insert(obj[3],-39088943,"Grigore")
TOPL: A new Treap was computed on the basis of an older-than-current one.
  -> start: return_insert(obj[0])
  -> start: return_insert(obj[0])
  -> start: call_insert(obj[0],"Dino")
  -> start: call_insert(obj[0],1018954901,"Dino")
  -> start: call_insert(obj[1],1018954901,"Dino")
  -> start: return_insert(obj[2])
  -> current: return_insert(obj[3])
  -> current: return_insert(obj[3])
  -> old: call_insert(obj[3],"Grigore")
  -> error: call_insert(obj[3],-39088943,"Grigore")
TOPL: A new Treap was computed on the basis of an older-than-current one.
  -> start: call_insert(obj[0],"Grigore")
  -> start: call_insert(obj[0],-39088943,"Grigore")
  -> start: call_insert(obj[1],-39088943,"Grigore")
  -> start: call_insert(obj[2],-39088943,"Grigore")
  -> start: return_insert(obj[3])
  -> start: return_insert(obj[4])
  -> start: return_insert(obj[5])
  -> current: return_insert(obj[5])
  -> old: call_insert(obj[5],"Distefano")
  -> error: call_insert(obj[5],1295249578,"Distefano")
TOPL: A new Treap was computed on the basis of an older-than-current one.
  -> start: call_insert(obj[0],"Grigore")
  -> start: call_insert(obj[0],-39088943,"Grigore")
  -> start: call_insert(obj[1],-39088943,"Grigore")
  -> start: call_insert(obj[2],-39088943,"Grigore")
  -> start: return_insert(obj[3])
  -> start: return_insert(obj[4])
  -> current: return_insert(obj[5])
  -> current: return_insert(obj[5])
  -> old: call_insert(obj[5],"Distefano")
  -> error: call_insert(obj[5],1295249578,"Distefano")
