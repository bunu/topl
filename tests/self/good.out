
[ Dino Distefano Grigore Radu ]
[ Dijkstra Dino Grigore Radu ]
